nbgitpuller/static/js/index.js

```diff
@@ -149,6 +153,7 @@ var gsv = new GitSyncView(
 );
 
 gs.addHandler('syncing', function(data) {
+    gsv.setTerminalVisibility(true);
```

Comment:
I'm not sure how this change related or motivated by this PR. Perhaps an inline comment would be relevant here.

Reply:
I made this change so users could see the progress automatically(or I could when developing!) I could take it out. The previous implementation forced the user to click the bar to open the terminal or if there was an error it would open automatically.